### PR TITLE
Fixed the MID collision in sgp4 and simple app.

### LINKED ISSUE #12: Fix MID collision with SGP4Prop example and CFE_Time
The SGP4 propagator example uses 0x1805 as it's house keeping message ID but this is the MID assigned to CFE_TIME_CMD_MID. To prevent future collisions with the cfe-core, we should switch all the message ID's in our examples to something like 0x[0|1]AA[x] to prevent accidentally assigning to another MID.

### PR DESCRIPTION
**Pull request by BaldBeacon**
close #12

### CODE REVIEW

File: examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
-#define OP_HK_MID     		0x1804     
-#define OP_PROPSTATE_MID 	0x1805
-
-#define ECI_TBL_MANAGE_MID 	0x1806
-#define ECI_SEND_HK_MID 	0x1807           
+#define OP_FLAG_MID         0x1AA0
+#define OP_CMD_MID          0x1AA1
+#define OP_PERF_ID          0x1AA2
+#define OP_TICK_MID         0x1AA3
+#define OP_HK_MID           0x1AA4
```
Comment (SpaceSteve121) [inline examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h]: It appears that this is the MID for the HK packet that ECI produces and therefore should have a TLM MID. Same comment applied for both examples.

File: examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h [hunk 2]
```diff
@@ -1,18 +1,14 @@
...
-#define OP_FLAG_MID 		0x1800
-#define OP_CMD_MID  		0x1801                  
-#define OP_PERF_ID   		0x1802            
-#define OP_TICK_MID   		0x1803              
-#define OP_HK_MID     		0x1804     
-#define OP_PROPSTATE_MID 	0x1805
-
-#define ECI_TBL_MANAGE_MID 	0x1806
-#define ECI_SEND_HK_MID 	0x1807           
+#define OP_FLAG_MID         0x1AA0
```
Comment (SpaceSteve121) [inline examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h]: Tlm?

File: examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h [hunk 3]
```diff
@@ -1,18 +1,14 @@
...
+#define OP_CMD_MID          0x1AA1
+#define OP_PERF_ID          0x1AA2
```
Comment (SpaceSteve121) [inline examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h]: It actually appears that this is not a MID, its a performance ID, so perhaps should be numbered separately from the sequence of APID's? Might be worth it to look for any standard that the CFE might follow for numbering these just to set a good example.

File: examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h (hunk 2 above)
Comment (BaldBeacon) [reply examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h]: Are you saying OP_FLAG_MID should be telemetry?

Comment (SpaceSteve121) [reply examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h]: For the SGP4 example, the [eci_interface.h](https://github.com/nasa/ECI/blob/cfe-6.6/examples/sgp4Prop/fsw/eci_inc/eci_interface.h#L23) specifies that OP_FLAG_MID is the value of ECI_FLAG_MID. ECI_FLAG_MID is used to [initialize ECI_AppData.FDCPacket](https://github.com/nasa/ECI/blob/cfe-6.6/fsw/src/eci_app.c#L693), which is defined as [a App_FaultRep_SbMsg type](https://github.com/nasa/ECI/blob/cfe-6.6/fsw/src/eci_app.c#L126) which is defined [here](https://github.com/nasa/ECI/blob/cfe-6.6/fsw/faultrep/app_faultrep.h#L139) as having a header `CFE_SB_TLM_HDR_SIZE` bytes long. So yea, looks like should be telemetry packet.

File: examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h (hunk 1 above)
Comment (BaldBeacon) [reply examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h]: Folded this into the new commits. I was more regimented about cmd/tlm designations.

File: examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h (hunk 2 above)
Comment (BaldBeacon) [reply examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h]: Folded this into the new commits. I was more regimented about cmd/tlm designations.

File: examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h (hunk 3 above)
Comment (BaldBeacon) [reply examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h]: Folded this into the new commits. I was more regimented about cmd/tlm designations.

**Comment by SpaceSteve121** [comment]
This looks fine. Did you try running these locally? Just because we don't yet have the tests which automatically startup the CFS.

Also, these changes also need to be made in the master. Want to submit the corresponding PR?

**Comment by BaldBeacon** [comment]
Yeah I ran these locally with both sgp4 and the simple app.

As for the changes to master, I'm waiting for this PR to get approved to make sure we agree on what needs to change.

**Review by SpaceSteve121 — APPROVED** [review]

**Pull request merged**